.circleci/config.yml

```diff
@@ -449,7 +449,10 @@ jobs:
     steps:
       - run-tests-linux:
           frozen_cache: false
-          test_targets: "wasm64.test_hello_world wasm64.test_ccall wasm64l"
+          test_targets: "wasm64"
+      - run-tests-linux:
+          frozen_cache: false
+          test_targets: "wasm64l"
```

Comment:
Do we need wasm64l now that we have full wasm64 up and running?

Reply:
Yes, I think wasm64l will always be useful won't it?  IIRC wouter designed to mode mostly so that 64-bit software could be run on wasm without requiring it to be ported to IPL32... In that use case such software still wants to target wasm32.. assuming it doesn't need more than 4Gb of memory.

Reply:
Hmm, true... but I thought the main use case was to bootstrap testing on wasm64, which is now complete basically. For the use case of helping people port 64-bit code to wasm32, I'm not sure it's worth us running all of `wasm64l` constantly here. We do test the lowering pass in Binaryen, and we could test just a select few tests here perhaps. That seems more efficient to me. But we don't need to decide on that urgently I guess.

Reply:
The other issue is that wasm64 is only currently runnable under v8..  we need to keep wasm64l around at least until we can run wam64 under node and in the browser.   Also, these tests really don't take that long to run since they are still a subset of `core`.   Each of these runs takes around 2 minutes in CI